### PR TITLE
[stable/atlantis] deprecating chart

### PR DESCRIPTION
**Pull request by torstenwalter**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #
  Part-of: #21103

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Review by scottrigby — APPROVED** [review]
💯

File: stable/atlantis/Chart.yaml [hunk 1]
```diff
@@ -1,17 +1,15 @@
 apiVersion: v1
+# The prometheus-operator chart is deprecated and no longer maintained. For details deprecation,
```
Comment (lkysow) [inline stable/atlantis/Chart.yaml:2]: @torstenwalter looks like you're setting "prometheus-operator" comments for charts that aren't the prometheus-operator.

Comment (torstenwalter) [reply stable/atlantis/Chart.yaml:2]: That was a copy paste mistake. Will create a PR to fix it.

Comment (torstenwalter) [reply stable/atlantis/Chart.yaml:2]: As promised https://github.com/helm/charts/pull/23855

**Pull request merged**